style.css

```diff
@@ -0,0 +1,494 @@
+}
+.menu li a:hover{
+    background: #0000007d;
+}
+.social-network{
+    display: flex;
+    flex-direction: column;
+}
+.social-network li{
+    list-style: none;
+}
+.social-network li img{
+    width: 50px;
+
+    filter: invert(1);
+}
+.social-network li img:hover{
+    filter: invert(0.1);
+}
+#openside:checked ~ .sidebar {
+    transform: translateX(0);
+}
+
+input#openside{
+    display: none;
+}
+.sidebaricon {
+    position: fixed;
+    height: 22px;
+    width: 22px;
+    top: 22px;
+    right: 40px;
+
+    transition: all 0.2s;
+
+    cursor: pointer; 
+    z-index: 99;
+}
+.spinner {
+    height: 5px;
+    width: 30px;
+
+    background-color: black;
+
+    transition: all 0.2s;
+}
+.spinner.mid ,
+.spinner.bottom{
+    margin-top: 3px;
+}
+#openside:checked ~ .sidebaricon > .spinner.mid {
+    opacity: 0;
+}
+#openside:checked ~ .sidebaricon > .spinner.top {
+    transform: rotate(135deg);
+    margin-top: 8px;
+}
+#openside:checked ~ .sidebaricon > .spinner.bottom {
+    transform: rotate(-135deg);
+    margin-top: -9px;
+}
+
+/* Slide show*/
+.slideshow{
+    position: relative;
+
+    width: 100%;
+    height: 470px;
+
+    overflow: hidden;
+}
+.slideshow .slider{
+    position:relative;
+
+    top: 0;
+    left: 0;
+    width: 400%;
+    height: 470px;
+
+    animation: animate 15s linear infinite;
+    animation-play-state: running;
+    cursor: pointer;
+}
+.slideshow .slider:hover{
+    animation-play-state: paused;
+}
+.slideshow .slider .slide{
+    position: relative;
+
+    float: left;
+    width: 25%;
+    height: 100%;
+    overflow: hidden;
+}
+.slideshow .slider .slide.one button{
+    position: absolute;
+
+    width: 400px;
+    height: 100px;
+    padding: 5px;
+
+    font-size: 30px;
+    font-weight: bolder;
+    background: transparent;
+
+    border: 4px solid;
+    inset: 300px;
+}
+.slideshow .slider .slide.one button:hover{
+    color: palevioletred;
+}
+.slideshow .slider .slide.four button{
+    position: absolute;
+
+    width: 400px;
+    height: 100px;
+    padding: 5px;
+
+    font-size: 30px;
+    font-weight: bolder;
+
+    background: transparent;
+
+    border: 4px solid;
+    inset: 300px 200px;
+}
+.slideshow .slider .slide.four button:hover{
+    color: palevioletred;
+}
+.slideshow .slider .slide.one{
+    background: url(./images/slide4.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+.slideshow .slider .slide.two{
+    background: url(./images/slide2.jpg);
+    background-size: cover;
+}
+.slideshow .slider .slide.three{
+    background: url(./images/slide3.jpg);
+    background-size: cover;
+}
+.slideshow .slider .slide.four{
+    background: url(./images/slide1.webp);
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+/*Shop*/
+.shop{
+    position: relative;
+
+    width: 100vw;
+
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+
+    overflow: hidden;
+}
+::-webkit-scrollbar{
+    display: none;
```

Comment:
any specific reason to hide this? You can also customize scrollbar's styling. It's pretty easy (Google it)

Reply:
nope i just dont want that bottom slidebar on my screen that's why i was hide it. .but now i comment it out.